### PR TITLE
[AN-4654] Downloaded video not in gallery

### PR DESCRIPTION
**Pull request by hakonbo**
#### APK
[Download build #8391](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8391/artifact/build/artifact/wire-dev-PR556-8391.apk)
[Download build #8401](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8401/artifact/build/artifact/wire-dev-PR556-8401.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/messages/controllers/MessageActionsController.scala [hunk 1]
```diff
@@ -197,6 +196,9 @@ class MessageActionsController(implicit injector: Injector, ctx: Context, ec: Ev
           def onLoaded(uri: Uri): Unit = {
             onAssetSaved ! asset
             Toast.makeText(context, com.waz.zclient.ui.R.string.content__file__action__save_completed, Toast.LENGTH_SHORT).show()
+            val intent: Intent = new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE)
+            intent.setData(uri)
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/messages/controllers/MessageActionsController.scala]: Not sure if you're aware of the `returning` util method we have:

```
import com.waz.utils.returning
...
context.sendBroadcast(returning(new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE))(_.setData(uri)))
``` 

new lines can make a little more readable, this just saves the hassle of coming up with new variable names, especially as the scope gets a little bigger

File: app/src/main/scala/com/waz/zclient/messages/controllers/MessageActionsController.scala [hunk 2]
```diff
@@ -197,6 +196,9 @@ class MessageActionsController(implicit injector: Injector, ctx: Context, ec: Ev
...
+            context.sendBroadcast(intent)
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/messages/controllers/MessageActionsController.scala]: Not sure if you're aware of the `returning` util method we have:

```
import com.waz.utils.returning
...
context.sendBroadcast(returning(new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE))(_.setData(uri)))
``` 

new lines can make a little more readable, this just saves the hassle of coming up with new variable names, especially as the scope gets a little bigger

Comment (hakonbo) [inline app/src/main/scala/com/waz/zclient/messages/controllers/MessageActionsController.scala]: Ah, nice... I'll update...

**Comment by hakonbo** [comment]
build again

**Comment by hakonbo** [comment]
build again

**Pull request merged**